Add a fourth solution for List Removals (#519)
cpinitiative/usaco-guide: solutions/CSES List Removals.mdx

		B.upd(ind,-1);
	}
}
```

## Solution 4

> Using a segment tree to store the number of elements present in each segment and then traversing in a binary-search-like manner.

Reviewer: ```suggestion
Use a segment tree to store the number of elements present in each segment and [walk down it](https://usaco.guide/plat/seg-ext#walking-on-a-segment-tree)
```
